extensions/interactions/ImageClickInput/directives/ImageClickInput.js

```diff
@@ -47,7 +48,32 @@ oppia.directive('oppiaInteractiveImageClickInput', [
               '/' + encodeURIComponent($scope.filepath)) : null);
           $scope.mouseX = 0;
           $scope.mouseY = 0;
-          $scope.currentlyHoveredRegions = [];
+          $scope.deactivate = false;
```

Comment:
$scope.interactionIsActive = true;

(what you have now is a verb, which seems to represent a function)

Reply:
Done